src/neuroglancer/datasource/brainmaps/backend.ts

```diff
@@ -207,6 +211,100 @@ export class MeshSource extends ParameterizedMeshSource<MeshSourceParameters> {
     return makeRequest(parameters['instance'], httpCall, cancellationToken)
         .then(response => decodeFragmentChunk(chunk, response));
   }
+
+  filterFragments(fragmentIds: string[]|null, clipBounds: Bounds) {
+    clipBounds;
+    if (!fragmentIds) {
+      return fragmentIds;
+    }
+
+    let filteredFragments = [];
+    for (let fragmentId of fragmentIds) {
+      // TODO(blakely): Hardcoded for now, remove when we can filter on the backend.
+      const fragmentSize = 500;
+      let fragmentBounds = this.getFragmentBounds(fragmentId, vec3.clone([fragmentSize, fragmentSize, fragmentSize]));  
+      if (this.boundsIntersect(fragmentBounds, clipBounds)) {
+        filteredFragments.push(fragmentId);
+      }      
+    }
+    
+    return filteredFragments;
+  }
+
+  private numBitsInPositiveValue(value: number) {
```

Comment:
This can be a free function rather than a method since it doesn't depend on `this`.